.appveyor.yml

```
@@ -81,7 +99,10 @@ build_script:
                     Write-Host ("{0}`n{1}`n" -f ('-'*60), "PACKAGE: $package")
                     $package_dir = ls -recurse | ? { $_.Name -eq "$package.nuspec"} | select -First 1 | % Directory
                     if (!$package_dir) { Write-Warning "Can't find package '$package'"; continue }
-                    pushd $package_dir; choco pack; Push-Package; popd
+                    pushd $package_dir
+                      if (Test-Path update.ps1 -ea 0) { ./update.ps1 }
+                      choco pack; Push-Package;
+                    popd
```

Is it worth hardening this a little, i.e. to ensure we end up back out of the $package_dir?

```suggestion
                    try {
                      pushd $package_dir
                      if (Test-Path update.ps1 -ea 0) { ./update.ps1 }
                      choco pack
                      Push-Package
                    } finally {
                      popd
                    }
```

---

I've added this, it seems like a good idea.